frontend/src/features/crawl-workflows/workflow-editor.ts

```diff
@@ -1366,6 +1343,38 @@ https://archiveweb.page/images/${"logo.svg"}`}
         ></btrix-select-browser-profile>
       `)}
       ${this.renderHelpTextCol(infoTextStrings["browserProfile"])}
+      ${inputCol(html`
+        <sl-radio-group
+          name="scale"
+          label=${msg("Browser Windows")}
+          value=${this.formState.scale}
+          @sl-change=${(e: Event) =>
+            this.updateFormState({
+              scale: +(e.target as SlCheckbox).value,
+            })}
+        >
+          ${when(this.appState.settings?.numBrowsers, (numBrowsers) =>
+            map(
+              range(this.defaults.maxScale),
+              (i: number) =>
+                html` <sl-radio-button value="${i + 1}" size="small"
+                  >${(i + 1) * numBrowsers}</sl-radio-button
+                >`,
+            ),
+          )}
+        </sl-radio-group>
+      `)}
+      ${this.renderHelpTextCol(
+        html`${msg(
+            `Increase the number of open browser windows during a crawl. This will speed up your crawl by effectively running more crawlers at the same time.`,
+          )}
+          <a
+            href="https://docs.browsertrix.com/user-guide/workflow-setup/#browser-windows"
+            class="text-blue-600 hover:text-blue-500"
+            target="_blank"
+            >${msg("See caveats")}</a
```

Comment:
Don't have to do it for this PR, but it would be _really cool_ if docs links we link to in the app opened up in the sidebar!